packages/flutter_test/lib/src/test_async_utils.dart

```diff
@@ -7,6 +7,9 @@ import 'dart:async';
 import 'package:flutter/foundation.dart';
 
 class _AsyncScope {
+  // This class is intended to be used as an interface, and should not be
+  // extended directly; this constructor prevents instantiation and extension.
+  // ignore: unused_element
```

Comment:
Why does this one need to be ignored? It's instantiated in line 70 below.

Reply:
Sorry, wrong class in this file. Fixed.